lang/cs/Org.Apache.REEF.Common/Files/PathUtilities.cs

```diff
@@ -45,7 +45,7 @@ public static string NormalizePath(string path)
         /// <returns></returns>
         public static bool AreNormalizedEquals(string path1, string path2)
         {
-            if (null == path1 || null == path2)
+            if (path1 == null || path2 == null)
```

Comment:
I think it is a good moment to rewrite the whole condition as
```C#
return path1 != null && path2 != null && NormalizePath(path1).Equals(NormalizePath(path2));
```

Reply:
Makes sense. Will do that in the next update.